src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer johnDeere = new Bulldozer("Name : John Deere 7930", " Color : Green",
+                220,
+                7.8,
+                390);
+        Bulldozer cat = new Bulldozer("Name : cat Chalenget MT865", "Color : Yellow",
```

Comment:
Consider removing the 'Name :' and 'Color :' prefixes from the bulldozer names and colors for consistency and clarity. The constructor parameters should directly represent the name and color values without additional text.